docs/synctl-create-test.md

```diff
@@ -21,7 +21,7 @@ synctl create test [options]
     --retries <int>                     retry times, value is from [0, 2]
     --retry-interval <int>              retry interval, range is [1, 10]
     --timeout <num>ms|s|m               set timeout, accept <number>(ms|s|m)
-    --custom-property <string>          An object with name/value pairs to provide additional information of the Synthetic test
+    --custom-properties <string>          An object with name/value pairs to provide additional information of the Synthetic test
```

Comment:
Adjust the format
<img width="441" alt="image" src="https://github.com/instana/synthetic-synctl/assets/25629629/99c4f706-86bf-4067-a575-0c654262dae1">